client/state/plugins/wporg/actions.js

```diff
@@ -2,18 +2,35 @@
  * External dependencies
  */
 const debug = require( 'debug' )( 'calypso:wporg-data:actions' );
-
+import debounce from 'lodash/debounce';
```

Comment:
we're using direct imports from `lodash` and a Babel transform minimizes them appropriately

```js
import { debounce } from 'lodash';
```